packages/react/src/components/Masthead/HeaderNavContainer.js

```diff
@@ -134,17 +134,21 @@ const HeaderNavContainer = ({ children }) => {
   }, [setIO]);
 
   useEffect(() => {
+    const navContent = contentRef.current;
+
     if (io) {
+      navContent.addEventListener('keydown', handleOnKeyDown);
       io.observe(contentLeftRef.current);
       io.observe(contentRightRef.current);
     } else {
       return () => {
         if (io) {
+          navContent.removeEventListener('keydown', handleOnKeyDown);
           io.disconnect();
         }
       };
     }
-  }, [io]);
+  });
```

Comment:
Seems that the change makes the callback run on every render, was this just for ESLint or of a bigger issue...?